src/librbd/librbd.cc

```diff
@@ -519,6 +520,30 @@ namespace librbd {
     return 0;
   }
 
+  int RBD::features_to_string(uint64_t features, std::string *str_features)
+  {
+    std::stringstream err;
+    *str_features = librbd::rbd_features_to_string(features, &err);
+    if (!err.str().empty())
+    {
```

Comment:
Nit: comment valid throughout but move starting curly bracket this to the previous line